src/v1/examples/LayeredPenguins.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {LPNClientV1} from "../client/LPNClientV1.sol";
+import {ILPNRegistryV1} from "../interfaces/ILPNRegistryV1.sol";
+import {
+    ERC721Enumerable,
+    ERC721
+} from "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import {L1BlockNumber} from "../../utils/L1Block.sol";
+import {QueryOutput} from "../Groth16VerifierExtensions.sol";
```

Comment:
Would be nice to be able to import only a single interface instead of 3 here. Even better if we could directly reference an URL or stg no ?